canonical-web-and-design/maas-ui
# refactor(ui): make the DHCP table work for all nodes

## Done

- Update the DHCP table to handle all node types.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to settings and add a DHCP snippet to a machine.
- Go to the network tab for the machine you added the DHCP snippet to.
- You should see the DHCP table at the bottom.
- You should be able to edit the DHCP snippet.

## Fixes

Fixes: canonical-web-and-design/app-tribe#568.
